src/stores/contactmoment/index.ts

```diff
@@ -47,6 +56,7 @@ export interface Vraag {
   werkinstructies: { werkinstructie: Werkinstructie; shouldStore: boolean }[];
   primaireVraag: { url: string; title: string } | undefined;
```

Comment:
dat PrimaireVraag type kan je hier ook voor gebruiken denk ik?

Reply:
ja, alleen is het geen primaire vraag. heb hem bron genoemd en ook voor een aantal andere bronnen gebruikt

Reply:
ja, alleen is het geen primaire vraag. heb hem bron genoemd en ook voor een aantal andere bronnen gebruikt